packages/google_sign_in/lib/src/device_flow_widget.dart

```diff
@@ -33,43 +33,102 @@ void showDeviceFlowWidget({
     context: navigatorKey.currentContext!,
     barrierDismissible: false,
     builder: (BuildContext context) {
+      final TextStyle bodyStyle =
+          Theme.of(context).textTheme.bodyLarge!.copyWith(color: Colors.black);
+      final TextStyle titleStyle = Theme.of(context)
+          .textTheme
+          .titleLarge!
+          .copyWith(color: Colors.black, fontWeight: FontWeight.bold);
+
       return AlertDialog(
-        title: const Text('Google Sign In'),
-        content: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: <Widget>[
-            const Text('From a PC, phone, or tablet go to:'),
-            Text(
-              '$verificationUrl',
-              textScaleFactor: 1.5,
-            ),
-            const Text('Then enter the code below:'),
-            Text(
-              code,
-              textScaleFactor: 2.5,
-            ),
-            const Text('Code will expire in:'),
-            _CountdownTimer(
-              expiresIn,
-              onFinished: () {
-                onExpired?.call();
-                closeDeviceFlowWidget();
-              },
-            ),
-          ],
+        scrollable: true,
+        contentPadding: const EdgeInsets.all(10),
+        content: Container(
+          padding: const EdgeInsets.symmetric(horizontal: 15, vertical: 25),
+          decoration: BoxDecoration(
+              border: Border.all(color: Colors.grey),
+              borderRadius: const BorderRadius.all(Radius.circular(5))),
```

Comment:
Missing trailing comma.

```suggestion
              borderRadius: const BorderRadius.all(Radius.circular(5)),),
```

Also you could remove the border itself (and padding) and save some space on devices with small screens.

Reply:
Done, I removed the border decoration and container padding but left the content padding for `AlertDialog`.